src/algebra/indicator_function.lean

```diff
@@ -159,6 +159,10 @@ s.apply_piecewise _ _ (λ _, h)
   mul_indicator (f ⁻¹' s) (g ∘ f) x = mul_indicator s g (f x) :=
 by { simp only [mul_indicator], split_ifs; refl }
 
+@[to_additive] lemma image_mul_indicator {s : set α} {f : β → M} {g : α → β} (hg : injective g)
```

Comment:
Perhaps the name `mul_indicator_image` would be better, since the lemma describes the `mul_indicator` of the image of a set, and not the image of something by `mul_indicator`?